nntrainer/include/tflite_layer.h

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Parichay Kapoor <pk.kapoor@samsung.com>
+ *
+ * @file	tflite_layer.h
+ * @date	3 November 2020
+ * @brief	This is class to encapsulate tflite as a layer of Neural Network
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Parichay Kapoor <pk.kapoor@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ */
+
+#ifndef __TENSORFLOW_LITE_H__
+#define __TENSORFLOW_LITE_H__
+#ifdef __cplusplus
+
+#include <layer_internal.h>
+#include <tensor.h>
+
+#include <tensorflow/contrib/lite/interpreter.h>
```

Comment:
Maybe later... How about using pimple idiom for this layer to isloate tflite dependency?

Reply:
I was thinking more along the lines of #152  because with pimpl, we still need to compile together.